content/servers.json5

```diff
@@ -49,10 +49,10 @@ links (array): objects representing game join links:
 		{
 			name: '/TG/Station 13',
 			logo: 'tgstation.png',
-			tags: ['experienced', 'lightrp'],
+			tags: ['experienced', 'lightrp', 'mediumrp', 'casual', 'expert'],
```

Comment:
it probably wants these alphabetical lol

Reply:
Nope. Site displays them in order of definition, and this order is fine.